tests/methods/mp2-nos-1/input.dat

```diff
@@ -1,5 +1,5 @@
 # Test RMP2 natural orbitals
-mp2_corr_ref = -1.802862266979274
+mp2_corr_ref = -1.80286362
```

Comment:
If you add `e_convergence 12` after `d_convergence 6` the energy will still match, but there will be a mismatch with the 1RDM. So the problem here is that the test is not well defined. So we need to ensure that SCF is converged tightly first by setting `e_convergence 12`.